airflow/providers/amazon/aws/utils/emailer.py

```diff
@@ -16,13 +16,15 @@
 # specific language governing permissions and limitations
 # under the License.
 """Airflow module for email backend using AWS SES"""
-
+from email.utils import formataddr
 from typing import List, Optional, Union
 
 from airflow.providers.amazon.aws.hooks.ses import SESHook
 
 
 def send_email(
+    from_email: str,
+    from_name: str,
```

Comment:
Not sure if we need to make these optional for backward compatibility. Does this function work before this PR?

Reply:
No, it was not working

Reply:
Cool.